src/main/java/jarvis/ui/TaskCard.java

```diff
@@ -26,11 +26,13 @@ public class TaskCard extends UiPart<Region> {
     @FXML
     private HBox cardPane;
     @FXML
+    private Label id;
+    @FXML
+    private Label done;
+    @FXML
     private Label taskDesc;
     @FXML
     private Label taskDeadline;
-    @FXML
-    private Label id;
 
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
```

Comment:
Spotted `PersonCode` and `Person` here, should be `TaskCode` and `Task`

Reply:
Fixed